scanner/src/main/kotlin/scanners/fossid/FossIdConfig.kt

```diff
@@ -80,6 +80,9 @@ internal data class FossIdConfig(
     /** A maximum number of delta scans to keep for a single repository. */
     val deltaScanLimit: Int,
 
+    /** Timeout in minutes for communication with FossID. */
+    val timeout: Long,
```

Comment:
Minor nit: As we now use minutes as unit for timeouts, this property should probably better of type `Int` rather than `Long`, since Java APIs typically use `Long` values to represent milliseconds. But I leave this to you.